src/components/main/Operation.js

```diff
@@ -0,0 +1,37 @@
+import './Operation.scss';
+
+import Button from '../elements/Button';
+import Wrapper from '../elements/Wrapper';
+import TodoLib from '../../store/TodoLib';
+
+const Operation = () => {
```

Comment:
- [ ] When removing completed tasks, the index is not correct after removing some items from the list, kindly make sure it updates all the items indexes so the updated array of items is stored in local storage with the correct list values after the page reloads

Reply:
@Ismailco Ok Thank You.